Source/Flow/Public/Nodes/FlowPin.h

```diff
@@ -21,6 +30,63 @@ struct FLOW_API FFlowPin
 	UPROPERTY(EditDefaultsOnly, Category = "FlowPin")
 	FString PinToolTip;
 
+protected:
+	// PinType (implies PinCategory)
+	UPROPERTY(EditAnywhere, Category = "FlowPin")
+	EFlowPinType PinType = EFlowPinType::Exec;
+
+	// Sub-category object
+	// (used to identify the struct or class type for some PinCategories, see IsSubtypeSupportedPinCategory)
+	UPROPERTY()
+	TWeakObjectPtr<UObject> PinSubCategoryObject;
+
+#if WITH_EDITORONLY_DATA
+#if 0 // TODO (gtaylor) Finish Class/Object support
+	// Filter for limiting the compatible classes for this data pin.
+	// This property is editor-only, but it is automatically copied into PinSubCategoryObject if the PinType matches (for runtime use).
+	UPROPERTY(EditAnywhere, Category = "FlowPin", meta = (EditCondition = "PinType == EFlowPinType::Class", EditConditionHides))
+	TSubclassOf<UClass> SubCategoryClassFilter;
+
+	// Filter for limiting the compatible object types for this data pin.
+	// This property is editor-only, but it is automatically copied into PinSubCategoryObject if the PinType matches (for runtime use).
+	UPROPERTY(EditAnywhere, Category = "FlowPin", meta = (EditCondition = "PinType == EFlowPinType::Object", EditConditionHides))
+	TSubclassOf<UObject> SubCategoryObjectFilter;
+#endif
+
+	// Configuration option for setting the EnumClass to a Blueprint Enum 
+	// (C++ enums must bind by name using SubCategoryEnumName, due to a limitation with UE's UEnum discovery).
+	// This property is editor-only, but it is automatically copied into PinSubCategoryObject if the PinType matches (for runtime use).
+	UPROPERTY(EditAnywhere, Category = "FlowPin", meta = (EditCondition = "PinType == EFlowPinType::Enum", EditConditionHides))
+	UEnum* SubCategoryEnumClass = nullptr;
+
+	// name of enum defined in c++ code, will take priority over asset from EnumType property
+	//  (this is a work-around because EnumClass cannot find C++ Enums, 
+	//   so you need to type the name of the enum in here, manually)
+	// See also: FFlowPin::PostEditChangedEnumName()
+	UPROPERTY(EditAnywhere, Category = "FlowPin", meta = (EditCondition = "PinType == EFlowPinType::Enum", EditConditionHides))
+	FString SubCategoryEnumName;
+#endif // WITH_EDITORONLY_DATA
+
+public:
+
+	// PinCategory aliases for (a subset of) those defined in UEdGraphSchema_K2
+	static inline FName PC_Exec = TEXT("exec");
```

Comment:
I aliased these (but made them cross-compatible with) UE's PC_* defines, which are editor-only.  But we need them at runtime as well.